.travis.yml

```diff
@@ -186,43 +193,40 @@ jobs:
     - stage: test
```

Comment:
It might be  a  good  idea to collapse the  operator scorecard  and molecule stages into  one to reduce the number of builds. Scorecard is so short that we don't get the  benefit of parallelism and it slows down the travis run,

Reply:
Opened an issue to track that, it's a good idea.